great_expectations/render/renderer/slack_renderer.py

```diff
@@ -103,56 +84,39 @@ def _build_title_block(
             },
         }
 
-    def _build_report_element_block(
-        self, data_docs_pages: list[dict], notify_with: list[str]
-    ) -> dict | None:
-        if not data_docs_pages:
-            return None
-
-        if notify_with:
-            for docs_link_key in notify_with:
-                if docs_link_key in data_docs_pages:
-                    docs_link = data_docs_pages[docs_link_key]
-                    report_element = self._get_report_element(docs_link)
-                else:
-                    logger.critical(
-                        f"*ERROR*: Slack is trying to provide a link to the following DataDocs: `"
-                        f"{docs_link_key!s}`, but it is not configured under `data_docs_sites` in the "  # noqa: E501
-                        f"`great_expectations.yml`\n"
-                    )
-                    report_element = {
-                        "type": "section",
-                        "text": {
-                            "type": "mrkdwn",
-                            "text": f"*ERROR*: Slack is trying to provide a link to the following DataDocs: "  # noqa: E501
-                            f"`{docs_link_key!s}`, but it is not configured under "
-                            f"`data_docs_sites` in the `great_expectations.yml`\n",
-                        },
-                    }
-                if report_element:
-                    return report_element
-        else:
-            for docs_link_key in data_docs_pages:
-                if docs_link_key == "class":
-                    continue
-                docs_link = data_docs_pages[docs_link_key]
-                report_element = self._get_report_element(docs_link)
-                return report_element
-
-        return None
-
-    def concatenate_text_blocks(self, text_blocks: list[dict], name: str, success: bool) -> dict:
-        all_blocks = [self._build_header(name=name, success=success), self._build_divider()]
+    def concatenate_text_blocks(  # noqa: PLR0913
+        self,
+        action_name: str,
+        text_blocks: list[dict],
+        success: bool,
+        checkpoint_name: str,
+        run_id: RunIdentifier,
+    ) -> dict:
+        all_blocks = [
+            self._build_header(name=action_name, success=success, checkpoint_name=checkpoint_name)
+        ]
+        all_blocks.append(self._build_run_time_block(run_id=run_id))
         for block in text_blocks:
             all_blocks.append(block)
-            all_blocks.append(self._build_divider())
-        all_blocks.append(self._build_footer())
+        all_blocks.append(self._build_divider())
 
-        return {"blocks": all_blocks, "text": name}
+        return {"blocks": all_blocks}
+
+    def _build_header(self, name: str, success: bool, checkpoint_name: str) -> dict:
+        status = "Success :white_check_mark:" if success else "Failure :no_entry:"
```

Comment:
Nit - could we move these Slack emojis to class vars? They're currently magic strings (I know I left it like this before)